src/components/ChatLog.jsx

```diff
@@ -0,0 +1,23 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({ entries, onToggleLike }) => {
+    const chatEntries = entries.map(entry => (
+        <ChatEntry
+            key={entry.id}
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            liked={entry.liked}
+            onToggleLike={onToggleLike}
+        />
+    ));
+
+    return (
+        <section className='chat-log'>{chatEntries}</section>
+    )
+
+};
+
+export default ChatLog;
```

Comment:
You didn’t include prop types for this component. While PropTypes is deprecated in React v19, it's still important to ensure consistent type safety across your application. Recall that React encourages you to adopt a type system like `TypeScript` to prevent type-related bugs and improve code reliability. @malikelmessiry